apps/reference/docs/guides/functions.mdx

```diff
@@ -270,5 +270,5 @@ serve(async (req) => {
   - Cannot write to File System
 - Edge Functions
   - Local development - only one function at a time
-  - Supabase Functions only supports `POST`, `OPTIONS`, and `GET` requests.
+  - Supabase Functions supports `GET`, `POST`, `PUT`, `PATCH`, `DELETE` and `OPTIONS` HTTP methods.
```

Comment:
On second thought @laktek - not that it supports everything it doesn't really make sense to have this in "Limitations". Should we remove this line completely?

Reply:
ah yeah, let's move this to a different section.